src/Core/IEnvironmentSettings.cs

```diff
@@ -35,7 +35,7 @@ public interface IEnvironmentSettings
         /// <summary>
         /// Returns the user-specific timeout, or zero if one has not been specified
         /// </summary>
-        int CFamilyAnalysisTimeoutInMs();
+        int AnalysisTimeoutInMs();
```

Comment:
Should I rename the env variable as well?
It's currently SONAR_INTERNAL_CFAMILY_ANALYSIS_TIMEOUT_MS

Reply:
Yes, we should. The only problem is that some users are using it is a workaround for analysis timeout issues: see https://community.sonarsource.com/t/crash-with-sonarlint-in-with-extended-timeout-for-large-c-projects/21861.

So we should mention it in the release notes and post a note to those community threads.

Reply:
FYI I've added a documentation ticket to the sprint: #1509

Reply:
Alternatively, we could support both for backwards compatibility. WDYT?

Reply:
I'd say not - that's why it has "INTERNAL" in the name, because it's not a publicly supported feature and we're not guaranteeing we'll continue to support it.
But that doesn't mean we shouldn't give a heads-up to those users we know were using the old variable.